apps/meteor/client/views/marketplace/AppsPage/AppsPage.tsx

```diff
@@ -1,69 +1,35 @@
-import { Button, ButtonGroup, Icon, Skeleton } from '@rocket.chat/fuselage';
-import { useRoute, useSetting, useMethod, useTranslation, useCurrentRoute, useRouteParameter } from '@rocket.chat/ui-contexts';
+import { useTranslation, useRouteParameter } from '@rocket.chat/ui-contexts';
 import type { ReactElement } from 'react';
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 
 import Page from '../../../components/Page';
+import MarketplaceHeaderComponent from '../components/MarketplaceHeaderComponent';
 import AppsPageContent from './AppsPageContent';
 
-type AppsPageProps = {
-	isMarketplace: boolean;
+type HeaderTitle = 'Explore' | 'Enterprise' | 'Installed' | 'Private Apps' | 'Apps';
+
+const getTitle = (context: string): HeaderTitle => {
+	switch (context) {
+		case 'explore':
+			return 'Explore';
+		case 'enterprise':
+			return 'Enterprise';
+		case 'installed':
+			return 'Installed';
+		case 'private':
+			return 'Private Apps';
+	}
+
+	return 'Apps';
```

Comment:
this is the case of using a translations, isnt it?

Reply:
Eeeeew, I forgot to change that one in the last moment... fixing right now!